src/Core/Merq/Merq.props

```diff
@@ -21,5 +26,13 @@
 			<PackageVersion>$(PackageVersion)$(VersionSuffix)</PackageVersion>
 		</PropertyGroup>
 	</Target>
+	
+	<Target Name="SetAssemblyVersion" BeforeTargets="GetAssemblyVersion" DependsOnTargets="GitInfo">
+		<PropertyGroup>
+			<AssemblyVersion Condition="'$(AssemblyVersion)' == ''">$(GitBaseVersionMajor).$(GitBaseVersionMinor).$(GitBaseVersionPatch)</AssemblyVersion>
```

Comment:
The SDK targets do the following:

```xml
  <Target Name="GetAssemblyVersion">
    <GetAssemblyVersion Condition="'$(AssemblyVersion)' == ''" NuGetVersion="$(Version)">
      <Output TaskParameter="AssemblyVersion" PropertyName="AssemblyVersion" />
    </GetAssemblyVersion>
    
    <PropertyGroup>
      <FileVersion Condition="'$(FileVersion)' == ''">$(AssemblyVersion)</FileVersion>
      <InformationalVersion Condition="'$(InformationalVersion)' == ''">$(Version)</InformationalVersion>
    </PropertyGroup>
  </Target>
```

Which means that both `$(AssemblyVersion)` and `$(FileVersion)` can be just inferred by setting `$(Version)` instead, since both are equal right now.